.travis.yml

```diff
@@ -15,13 +15,9 @@ matrix:
     - python: 3.6
       env: TOX_ENV=isort
     - python: 3.6
-      env: TOX_ENV=py36-django-111
-    - python: 3.6
-      env: TOX_ENV=py36-django-19
-    - python: 3.5
-      env: TOX_ENV=py35-django-19
-    - python: 3.4
-      env: TOX_ENV=py34-django-19
+      env: TOX_ENV=py36
+    - python: 3.7
+      env: TOX_ENV=py37
```

Comment:
can we support both Django 1.9 *and* 1.11?

Reply:
This will allow us to merge this now and use it in the current version of docs italia

Reply:
Done 👍

Reply:
:bowing_man: